ios/KlaviyoBridge.swift

```diff
@@ -82,6 +82,11 @@ public class KlaviyoBridge: NSObject {
       KlaviyoSDK().set(profile: profile)
   }
 
+  @objc
+  public static func setProfileAttribute(_ key: String, value: String) {
+      KlaviyoSDK().set(profileAttribute: getProfileKey(key), value: value)
```

Comment:
Doesn't really matter, but in the Android code we just used CUSTOM for all the keys 🤷 
```

    @ReactMethod
    override fun setProfileAttribute(
      propertyKey: String,
      value: String,
    ) {
      Klaviyo.setProfileAttribute(ProfileKey.CUSTOM(propertyKey), value)
    }
```